schema/collection.js

```diff
@@ -59,6 +59,19 @@ const CollectionType = new GraphQLObjectType({
   },
 })
 
+// This resolver is re-used by `me { saved_artworks }`
```

Comment:
so I think there's value, in that if others want to keep collections - this is the API they'd use